src/components/index.jsx

```diff
@@ -2,22 +2,32 @@ import Archive from "./Archive/Archive";
 import ColorPallete from "./ColorPallete/ColorPallete";
 import Editor from "./Editor/Editor";
 import EmptyState from "./EmptyState/EmptyState";
+import SortSelector from "./SortSelector/SortSelector";
 import Header from "./Header/Header";
 import HomePage from "./HomePage/HomePage";
 import LabelSelector from "./LabelSelector/LabelSelector";
-import NoteCard from "./Note/Note";
+import NoteCard from "./NoteCard/Note";
+import PrioritySelector from "./PrioritySelector/PrioritySelector";
 import Toast from "./Toast/Toast";
 import Trash from "./Trash/Trash";
+import TrashNote from "./TrashNote/TrashNote";
+import FilterSelector from "./FilterSelector/FilterSelector";
+import ArchiveNote from "./ArchiveNote/ArchiveNote";
```

Comment:
Instead of importing the components and then exporting them from your index.jsx,you can directly export them
```
export {TrashNote} from "./TrashNote/TrashNote"
export {FilterSelector} from "./FilterSelector/FilterSelector"
```
Don't know if it work with Default Exports or not. But I guess it shall work!!
Thus reducing your effort here!!

Reply:
Yes that makes sense, but the purpose of using an index file was so that I can import all my components in a single line